index.html

```diff
@@ -2975,45 +2976,38 @@ <h3>Zero-Knowledge Proofs</h3>
         </ul>
 
         <p>
-This specification describes a data model that supports zero-knowledge proof
-mechanisms. The examples below highlight how the data model can be used to
-issue, present, and verify zero-knowledge <a>verifiable credentials</a>.
+This specification describes a data model that supports selective disclosure
+with the use of a zero-knowledge proof mechanisms. The examples below highlight
+how the data model can be used to issue, present, and verify zero-knowledge
+<a>verifiable credentials</a>.
         </p>
 
         <p>
 To use zero-knowledge <a>verifiable credentials</a> the <a>issuer</a> must
-issue a <a>verifiable credential</a> in a manner that enables the <a>holder</a>
-to present the information to a <a>verifier</a> in a privacy-enhancing manner.
+issue a <a>verifiable credential</a> in a manner that enables
+the <a>holder</a> to derive a proof from the original issued <a>verifiable
+credential</a> so that the <a>holder</a> can present the information to a
```

Comment:
```suggestion
the <a>holder</a> to derive a proof from the originally issued <a>verifiable
credential</a>, so that the <a>holder</a> can present the information to a
```